app/views/pages/get-started/contact-us-tutorial/index.liquid

```diff
@@ -0,0 +1,168 @@
+
+### Step 3: Check if you have platformOS check and pos-cli installed
+
+Check if you have platformOS check and pos-cli installed:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+platformos-check -v
+pos-cli -v
+</code></pre>
+
+{% include 'alert/note', content: 'On Windows, you do not need to install `platformos-check` separately, as it is included in our [VS Code extension](https://marketplace.visualstudio.com/items?itemName=platformOS.platformos-check-vscode).' %}
+
+If you haven't installed pos-cli, you can do so globally using npm:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+npm install -g @platformos/pos-cli
+</code></pre>
+
+
+For further details, refer to the [Installation and Configuration guide](/get-started/installation-and-configuration/), the [Install pos-cli guide](/get-started/installation-and-configuration/#install-pos-cli),  and the [platformos-check guide](https://github.com/Platform-OS/platformos-check).
+
+
+### Step 4: Create the project folder
+
+Create the directory for the Contact Us project on your machine:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+mkdir contact-us
+cd contact-us
+</code></pre>
+
+{% include 'alert/note', content: 'This directory will be considered the **project root**, and all future commands, such as `pos-cli sync` and `pos-cli env add`, will need to be executed from this directory. If you encounter any errors with `pos-cli` about missing environments or other issues, the first thing to check is whether the command was executed from the project root.' %}
+
+
+### Step 5: Authenticate your environment
+
+For pos-cli to know which instance is related to any particular codebase, you need to make a connection between those. Authenticate your environment:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli env add --url https://YOUR-INSTANCE.staging.oregon.platform-os.com staging
+</code></pre>
+
+A hidden `.pos` file has been created, and this is what pos-cli uses behind the scenes to authenticate.
+
+For further details, refer to the [Authenticate your environment](get-started/working-with-the-code-and-files/#authenticate-your-environment) guide.
+
+### Step 6: Authorize your instance
+
+Authorize your instance in the browser when prompted:
+
+{% # TODO: check size %}
+<img loading="lazy" class="w-full" src="{{ 'images/get-started/contact-us-tutorial/authorize.png' | asset_url }}" alt="Authorize button on the Partnerl Portal">
+
+You should receive the following message:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+Environment https://contactus.staging.oregon.platform-os.com/ as staging has been added successfully.
+</code></pre>
+
```

Comment:
I think we are missing a step here when one uses `pos-cli init` because we are currently installing modules in an empty folder.

Reply:
@z-x I think I've fixed this https://github.com/Platform-OS/pos-cli/pull/625 back at a day :)